package.json missing name

I'm unable to install the v10.0.0 typings via npm due to the package.json file not having `name` property.

```
$ npm install --save git+https://github.com/zajrik/discord.js-typings.git#de59a23685c90429719111a4d5b7bd1cf57b2031
...
npm ERR! No name provided in package.json
...
```

--- a/package.json
+++ b/package.json
@@ -1,4 +1,7 @@
 {
+  "name": "discord.js-typings",
+  "version": "12.0.0",
+  "description": "Typings for Discord.js",
   "devDependencies": {
     "tslint": "^3.15.1",
     "tslint-config-typings": "^0.2.4",

